MAUI/ListView/viewappearance.md

```diff
@@ -499,6 +499,116 @@ public partial class MainPage : ContentPage
 
 ![MAUI ListView Drop shadow effect on items](Images/appearance/maui-listview-drop-shadow-effects-on-items.jpg)
 
+## Visual State Manager
+
+Use the visual state manager to change the .NET MAUI SfListView mouse hover background on the listView item based on the visual states set from code. The applicable visual states are Normal and PointerOver.
```

Comment:
Use the visual state manager to change the .NET MAUI SfListView mouse hover background on the listView item based on the visual states set from the code. The applicable visual states are Normal and PointerOver.

Reply:
resolved